README.md

```diff
@@ -83,6 +84,42 @@ GLHF! <3
 
 GLHF! <3
 
+## Tantangan
+
+Anda mempunyai satu bulan penuh pada bulan November untuk membuat sebuah permainan *bebas* dengan bertemakan **peretasan, modifikasi dan/atau penggandaan**.
+
+Apa yang kita maksud dengan **bebas** berdasarkan pada peretasan, modifikasi dan/atau penggandaan? Kami sebenarnya bermaksud, *bebas* yang disarankan. Beberapa contoh mungkin seperti ini:
+
+* seorang pelari yang tak berujung dimana anda *retas* pohon biner menurun di jalur anda dengan kapak yang berpiksel,
+* sebuah pengambilan secara *modern* pada sesuatu yang klasik misalnya sebuah roguelike yang diatur di dalam dunia 3D atau dunia VR, atau
+* sebuah *penggandaan* permainan yang nyata membawa gabungan gurita/kucing kedalam dunia nyata.
+
+Tunjukan kreatifitasmu. anda dapat bekerja sendiri atau dengan tim dan membangun untuk berbagai platform ataupun perangkat. Penggunaan mesin game dari sumber terbuka dan berbagai pustaka itu dianjurkan tetapi tidak diperlukan.
+
+Kita akan tetap memantau beberapa blog GitHub favorit kami, dan dunia dapat menikmati (dan bahkan mungkin berkontribusi dan/atau untuk belajar) dengan kreasi anda.
+
+## Bagaimana cara berpartisipasi
+
+* [Daftarkan untuk akun pribadi gratis][github-signup] jika anda belum mempunyainya.
+* Fork [repositori ini][game-off-repo] untuk akun pribadi (atau untuk sebuah [akun organisasi gratis][github-signup-org]).
+* Clone repositori pada komputer anda dan buat permainan anda.
+* Push kode sumber(source code) permainan anda untuk repositori hasil fork anda sebelum tanggal 1 Desember.
+* Perbaharui berkas `README.md` untuk memasukkan penjelasan dari permainan anda, bagaimana/dimana untuk memainkan/mengunduhnya, bagaimana cara build/compile permainan anda, ketergantungan apa yang dimiliki, dan lain-lain.
+* Ajukan versi terakhir permainan anda menggunakan [formulir][wufoo-form] ini.
+
+## Ini berbahaya untuk dilakukan sendiri <img src="https://octodex.github.com/images/linktocat.jpg" height="40">
+
+Jika anda masih **pemula dalam menggunakan Git, GitHub, atau version control**…
```

Comment:
_Version control_ can be referred as _revision control_, thus you can translate it as _pengendali revisi_ or _manajemen revisi_.